tox.ini

```diff
@@ -1,15 +1,18 @@
 [tox]
-envlist = py27
+envlist = tests, pylama
 
-[testenv:py27]
-deps =
-    mock
-    mozfile >= 1.1
-    mozprocess >= 0.22
-    pylama
-    pytest
-    wptserve >= 1.4.0
+[testenv:tests]
+usedevelop = true
+passenv = TRAVIS TRAVIS_JOB_ID TRAVIS_BRANCH
+deps = -rrequirements/tests.txt
```

Comment:
I think it's still worth having `coverage.txt` as users running tests locally would avoid the extra dependencies for coverage.